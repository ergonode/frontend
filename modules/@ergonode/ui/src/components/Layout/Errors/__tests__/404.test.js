/*
 * Copyright Â© Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE for license details.
 */
import Error404 from '@UI/components/Layout/Errors/404';
import {
    shallowMount,
} from '@vue/test-utils';

describe('Errors/404', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallowMount(Error404);
    });

    it('Component is named well', () => {
        expect(typeof Error404.name).toBe('string');
        expect(Error404.name).toEqual('Error404');
        expect(wrapper).toMatchSnapshot();
    });
});
