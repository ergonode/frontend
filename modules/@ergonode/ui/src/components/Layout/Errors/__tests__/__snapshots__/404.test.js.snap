// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors/404 Component is named well 1`] = `
<div class="placeholder">
  <div class="horizontal-wrapper"><img src="@UI/assets/images/placeholders/error-pages/404.svg" class="image">
    <div class="navigation-section">
      <h1 class="navigation-section__header">
        Not found
      </h1>
      <p class="navigation-section__description">
        The link you clicked may be broken or the page may have been removed.
      </p>
      <button-stub theme="primary" size="regular" title="Go to dashboard"></button-stub>
    </div>
  </div> <span class="error-section section__quotation">
        The page you were looking for doesnâ€™t exist. If you think this is an
        <br> error, please report it to the system administrator.
    </span>
</div>
`;
